app/v1/models/request/deleteSEAnnual/DeleteAnnualSubmissionRequest.scala

```diff
@@ -16,6 +16,6 @@
 
 package v1.models.request.deleteSEAnnual
```

Comment:
shall we change the package name to **deleteSEAnnual** -> **deleteAnnual**
same with other packages